src/app/global/local_app_settings.nim

```diff
@@ -13,6 +13,10 @@ const LAS_KEY_APP_HEIGHT = "global/app_height"
 const DEFAULT_APP_HEIGHT = 770
 const LAS_KEY_APP_SIZE_INITIALIZED = "global/app_size_initialized"
 const DEFAULT_APP_SIZE_INITIALIZED = false
+const LAS_KEY_IS_DDMMYY_DATE_FORMAT = "global/is_DDMMYY_date_format"
```

Comment:
@noeliaSD indeed these properties should be added to `local_account_sensitive_settings`, please move them there.